.github/workflows/build.yml

```diff
@@ -7,13 +7,13 @@ on:
   pull_request:
     types: [assigned, opened, synchronize, reopened]
     paths-ignore:
-      - '.husky/**'
-      - 'stories/**'
-      - '.vscode/**'
-      - 'README.md'
-      - 'CONTRIBUTING.md'
-      - 'CHANGELOG.md'
-      - 'LICENSE'
+      - ".husky/**"
```

Comment:
Maybe good to add rules to vscode settings or similar to avoid editor refactor quotes